model/gen-ai/registry.yaml

```diff
@@ -1,6 +1,7 @@
 groups:
```

Comment:
Generally we need to discuss and figure out how to draw the line between generic agent semantics vs vendor specific ones for different frameworks

Reply:
@karthikscale3 +1, we can probably follow same pattern as LLM, we can have a common agent semantic convention, and each agent framework can extend based on the common agent semantic convention

Reply:
rule of 3 is bare minimum on specs IMHO (and I know nobody asked me). If a spec doesn't cover 3 diverse impls, it will break until then (probably still break after though ;))

Reply:
@codefromthecrypt +1, that is why I did some research work at https://gyliu513.github.io/jekyll/update/2024/12/17/battle-of-ai-agent-frameworks.html for different agent frameworks, trying to summary them now.